hubspot/api_test.py

```diff
@@ -89,12 +103,79 @@ def test_make_contact_sync_message(user):
     serialized_user.update(serialized_user.pop("profile") or {})
     serialized_user["street_address"] = "\n".join(serialized_user.pop("street_address"))
 
-    time = contact_sync_message["changeOccurredTimestamp"]
-    assert contact_sync_message == (
+    time = contact_sync_message[0]["changeOccurredTimestamp"]
+    assert contact_sync_message == [
         {
             "integratorObjectId": str(user.id),
             "action": "UPSERT",
             "changeOccurredTimestamp": time,
             "propertyNameToValues": serialized_user,
         }
-    )
+    ]
+
+
+@pytest.mark.django_db
+def test_make_deal_sync_message():
+    """Test make_deal_sync_message serializes a deal and returns a properly formatted sync message"""
+    order = OrderFactory()
+    product_version = ProductVersionFactory()
+    company = CompanyFactory()
+    payment_version = CouponPaymentVersionFactory(company=company)
+    coupon_version = CouponVersionFactory(payment_version=payment_version)
+    LineFactory(order=order, product_version=product_version)
+    CouponRedemptionFactory(order=order, coupon_version=coupon_version)
+
```

Comment:
The above code is used twice, here and in `hubspot.api_test.test_make_deal_sync_message`.

Can you add this fixture to `hubspot/conftest.py`:
```python
@pytest.fixture
def hubspot_order():
    """ Return an order for testing with hubspot"""
    order = factories.OrderFactory()
    product_version = factories.ProductVersionFactory()
    company = factories.CompanyFactory()
    payment_version = factories.CouponPaymentVersionFactory(company=company)
    coupon_version = factories.CouponVersionFactory(payment_version=payment_version)
    factories.LineFactory(order=order, product_version=product_version)
    factories.CouponRedemptionFactory(order=order, coupon_version=coupon_version)
    return order
```

Then used that fixture in this test:
```pytest
def test_sync_deal_with_hubspot(mock_hubspot_request, hubspot_order):
    sync_deal_with_hubspot(hubspot_order.id)
    ...etc
```

and ditto for `hubspot.api_test.test_make_deal_sync_message`